src/app/core/models/rules-interfaces.model.ts

```diff
@@ -0,0 +1,29 @@
+export class Rule{
+  name: string = '';
+  id?: number;
+  created_at?: Date;
+  tardiness_rules_attributes?:TradinessRule[] = [];
+  deleted_tardiness_rules?: TradinessRule[] = [];
+  leave_type_id?: number;
+  is_half_day?: boolean;
+  }
+
+ export class TradinessRule{
```

Comment:
Please rename it to all **TardinessRule** and apply the change in all files.

Reply:
done